src/localsocketcontroller.cpp

```diff
@@ -87,12 +85,17 @@ void LocalSocketController::initialize(const Device* device, const Keys* keys) {
   Q_UNUSED(keys);
 
   Q_ASSERT(m_daemonState == eUnknown);
-  m_initializingRetry = 0;
 
   initializeInternal();
 }
 
 void LocalSocketController::initializeInternal() {
+  // Perform an exponential backoff.
```

Comment:
A comment that describes this design to detect a non-responding daemon would be useful here or in the header file.

Reply:
Done!